.coafile

```diff
@@ -1,10 +1,10 @@
-[Default]
-files = *.py, coala_langserver/*.py, tests/**/*.py
+[all]
```

Comment:
Is this change *necessary* as part of this PR?

You have two extremely large PRs doing all sorts of things, with commits that are not atomic, and even are regressions.

They are the anti-thesis on how to get a maintainer to merge your work quickly.
You had the same problem with https://github.com/coala/coala-vs-code/pull/32 - the same principles apply here.

Reply:
I am sorry for repeating that again. I have undone some of such changes and have divided the PR into more number of responsible commits.